runtimes/r/R/runtime.R

```diff
@@ -363,36 +363,6 @@ catala_assertion_failure <- function(pos) {
 
 ################ Defaults #################
 
-#' @export
-catala_handle_default <- function(pos, exceptions, just, cons) {
```

Comment:
Shouldn't that be replaced with a similar function to handle exceptions with options?

Reply:
Indeed, that's a "TODO" in the `to_r` module that remains for later